appveyor.yml

```diff
@@ -4,8 +4,7 @@ os: Visual Studio 2017 # Windows Server 2016
 install:
   - java -version
   - gradlew.bat --version
-build_script: gradlew.bat -u -i install
-test_script: gradlew.bat -u -i -S check
+build_script: gradlew.bat -u -i -S build # The 'build' task runs everything, including test, check, & static analysis
```

Comment:
✅ Combining the `install` and `check` steps is okay.
🎨 The tests are conventionally run in the `test_script` phase, rather than in the `build_script` phase. 

⬇️ Below is one way to match all the above comments:

``` yaml
environment:
  JAVA_HOME: C:\Program Files\Java\jdk1.8.0
os: Visual Studio 2017 # Windows Server 2016
install:
  - java -version
  - gradlew.bat --version
# The 'build' task runs everything, including test, check, & static analysis
build: off
test_script: gradlew.bat -u -i -S build
cache:
  - .gradle
  - C:\Users\appveyor\.gradle